route53_crossfeed_app.tf

```
@@ -271,57 +300,76 @@ resource "aws_route53_record" "crossfeed_staging_TXT" {
 # Staging API entries
 # ------------------------------------------------------------------------------
 
-resource "aws_route53_record" "crossfeed_staging_api_A" {
+resource "aws_route53_record" "crossfeed_staging_cd_api_A" {
   provider = aws.route53resourcechange
 
   alias {
     name                   = "d2n7pvqicqw5zg.cloudfront.net."
     zone_id                = var.cloudfront_zone_id
     evaluate_target_health = false
   }
-  name    = "api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  name    = "api.staging-cd.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
   type    = "A"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_AAAA" {
+resource "aws_route53_record" "crossfeed_staging_cd_api_AAAA" {
   provider = aws.route53resourcechange
 
   alias {
     name                   = "d2n7pvqicqw5zg.cloudfront.net."
     zone_id                = var.cloudfront_zone_id
     evaluate_target_health = false
   }
-  name    = "api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  name    = "api.staging-cd.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
   type    = "AAAA"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_CNAME1" {
+# The hosted_zone_id for the below records comes from https://docs.aws.amazon.com/general/latest/gr/elb.html
+# (ALBs in us-gov-west-1 region)
+resource "aws_route53_record" "crossfeed_staging_api_A" {
   provider = aws.route53resourcechange
 
-  name    = "_1cd918a94c59cbd2fdce9df0fdfc980f.api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
-  records = ["_f1fe0d0997fac6b9d221c8b482ac0ec2.jfrzftwwjs.acm-validations.aws."]
-  ttl     = 300
-  type    = "CNAME"
+  alias {
+    name                   = "crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com."
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+  name    = "api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  type    = "A"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_api_CNAME2" {
+resource "aws_route53_record" "crossfeed_staging_api_AAAA" {
   provider = aws.route53resourcechange
 
-  name    = "staging-cd.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
-  records = ["_73aacddb56ac20ef77492c962bf006d8.jfrzftwwjs.acm-validations.aws"]
+  alias {
+    name                   = "crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com."
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+  name    = "api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  type    = "AAAA"
+  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
+}
+
+resource "aws_route53_record" "crossfeed_staging_api_CNAME" {
```

Now that this has all been sorted out, if there's more than one CNAME record, the resource names should be CNAME1, CNAME2, etc:
```suggestion
resource "aws_route53_record" "crossfeed_staging_api_CNAME1" {
```